warehouse/models/staging/audit/stg_audit__cloudaudit_googleapis_com_data_access.sql

```diff
@@ -0,0 +1,95 @@
+{{
+    config(
+        materialized='incremental',
+        incremental_strategy='insert_overwrite',
+        partition_by={
+            'field': 'date',
+            'data_type': 'date',
+            'granularity': 'day',
+        },
+        partitions=['current_date()'],
+        cluster_by='job_type',
+    )
+}}
+-- we should use https://cloud.google.com/bigquery/docs/reference/standard-sql/json_functions#parse_json when available
+
+-- Note these two source CTEs a direct reference instead of source, because a new table is created daily
```

Comment:
incredible nitpick: I think this comment is missing a word? 

```suggestion
-- Note these two source CTEs use a direct reference instead of source, because a new table is created daily
```